financial/present_value.py

```diff
@@ -0,0 +1,44 @@
+# Reference: https://www.investopedia.com/terms/p/presentvalue.asp
+
+# Algorithm that calculates the present value of a stream of yearly cash flows given...
+# 1. The discount rate (as a decimal, not a percent)
+# 2. An array of cash flows, with the index of the cash flow being the associated year
+
+# Note: This algorithm assumes that cash flows are paid at the end of the specified year
```

Comment:
Docstrings are better because they will appear in automated documentation like Sphinx, etc.
```suggestion
"""
Reference: https://www.investopedia.com/terms/p/presentvalue.asp

An algorithm that calculates the present value of a stream of yearly cash flows given...
1. The discount rate (as a decimal, not a percent)
2. An array of cash flows, with the index of the cash flow being the associated year

Note: This algorithm assumes that cash flows are paid at the end of the specified year
```